RobotCode/src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.constants;
+
+public class ElevatorConstants {
+	
+	public static final double[] ELEVATOR_POSITIONS = { // In encoder ticks (2048 ticks = 1 deg)
```

Comment:
The units for a TalonFX are actually rotations (come ask me if you want an explanation of how ticks are related to degrees and rotations)